packages/core/botpress/src/web/views/GhostContent/index.js

```diff
@@ -79,17 +80,17 @@ export default class GhostView extends Component {
       }
     }
 
-    const undoLink = (
-      <a href="javascript: void(0);" onClick={undo}>
-        Revert changes
-      </a>
-    )
-
     return (
       <li key={`${folder}/${file}`}>
         <details>
           <summary>
-            {file} {undoLink}
+            {file}
+            <PermissionsChecker user={this.props.user} op="write" res="bot.ghost_content">
+              {' '}
```

Comment:
@emirotin , is it here for reason?

Reply:
It's the way to put space in JSX, could also be `&nbsp;`
It's often autoinserted by prettier when it splits the lines

Reply:
I see. Yes, the space is needed there